src/Microsoft.Sbom.Parsers.Spdx30SbomParser/Generator.cs

```diff
@@ -601,6 +595,28 @@ private Element GenerateLicenseElement(string licenseInfo)
         return licenseElement;
     }
 
+    private SpdxEntities.Relationship GetSpdxRelationship(string sourceElement, string targetElement, SbomEntities.RelationshipType relationshipType)
+    {
+        var spdxRelationshipType = this.GetSPDXRelationshipType(relationshipType);
+
+        // Switch source and target IDs for these specific relationship types
+        if (relationshipType == SbomEntities.RelationshipType.PREREQUISITE_FOR ||
+            relationshipType == SbomEntities.RelationshipType.DESCRIBED_BY ||
+            relationshipType == SbomEntities.RelationshipType.PATCH_FOR)
+        {
+            (sourceElement, targetElement) = (targetElement, sourceElement);
```

Comment:
Do/should we have unit tests for the `RelationshipType` values that don't change the relationship direction?

Reply:
Yup there's multiple other relationship types that do not change direction. There's already 1 UT that's representative of all of them here - GenerateJsonDocumentTest_Relationship